BaseFilter/sections/foreign.txt

```diff
@@ -4788,8 +4788,6 @@ tv.kakao.com,namu.wiki,daum.net#@##adContainer
 @@||videoads.kakao.com/adserver/api/*/vmap$domain=tv.kakao.com|namu.wiki|daum.net
 @@||videoads.kakao.com/adserver/api/*/vast$domain=tv.kakao.com|namu.wiki|daum.net
 !#endif
-! https://github.com/AdguardTeam/AdguardFilters/issues/78474
-dajunbi.com##.basic-banner
```

Comment:
Don't remove this rule, it's still relevant.

Reply:
Btw, the original rule
```
joytv.fun,dajunbi.com##.basic-banner
```

Reply:
Ok, what about this site `joytv.fun`? Is it opened on your end?

Reply:
`joytv.fun` is a dead domain.

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/5a7c269a72daf1279afa944c9a250128125159d1

Reply:
I can't, that's why I asked.